web/client/src/entities/CallForwardSetting/Field/CallForwardType.tsx

```diff
@@ -0,0 +1,56 @@
+import { ListDecorator, ScalarProperty } from '@irontec/ivoz-ui';
+import {
+  PropertyCustomFunctionComponent,
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui/services/form/Field/CustomComponentWrapper';
+import { useStoreState } from 'store';
+import { CallForwardSettingPropertyList } from '../CallForwardSettingProperties';
+
+type CallForwardTypeValues = CallForwardSettingPropertyList<string>;
+type CallForwardTypeProps = PropertyCustomFunctionComponent<PropertyCustomFunctionComponentProps<CallForwardTypeValues>>;
+
+const CallForwardType: CallForwardTypeProps = (props): JSX.Element | null => {
+
+  const { _context, _columnName, property, values, formFieldFactory } = props;
+  const aboutMe = useStoreState((state) => state.clientSession.aboutMe.profile);
+
+  if (_context === 'read' || !formFieldFactory) {
+    return (
+      <ListDecorator field={_columnName} row={values} property={property} ignoreCustomComponent={true} />
+    );
+  }
+
+  const { choices, readOnly } = props;
+  const modifiedProperty = { ...property } as ScalarProperty;
+  delete modifiedProperty.component;
+
+  if (!aboutMe) {
+    return formFieldFactory.getInputField(
+      _columnName,
+      modifiedProperty,
+      choices,
+      readOnly,
+    );
+  }
+
+  const enumValues = {
+    ...modifiedProperty.enum,
+  };
+
+  if (!aboutMe?.retail) {
+    delete enumValues.cfwToRetailAccount;
+    delete enumValues.ddi;
```

Comment:
These are not enum values from callForwardType

vPBX and Residential Call Forwards should not delete any values in this enum, so you can remove the upper part of the if.